waspc/cli/exe/Main.hs

```diff
@@ -95,49 +47,28 @@ main = withUtf8 . (`E.catch` handleInternalErrors) $ do
     handleInternalErrors :: E.ErrorCall -> IO ()
     handleInternalErrors e = putStrLn $ "\nInternal Wasp error (bug in compiler):\n" ++ indent 2 (show e)
 
-printUsage :: IO ()
```

Comment:
I also found this conversation, maybe it is useful: https://github.com/pcapriotti/optparse-applicative/issues/88 .

Might also be worth asking directly on GH repo of optparse (opening an issue), and/or asking on r/haskell.

Reply:
And this section in their README is what we care about: https://github.com/pcapriotti/optparse-applicative#customising-the-help-screen . But it doesn't say much about stuff like coloring hm.

Reply:
I've opened an issue in optparse-applicative repo https://github.com/pcapriotti/optparse-applicative/issues/478. One of the maintainers has replied the issue. I don't think it's possible at the moment. Hmm. :thinking: :crying_cat_face:. Maybe I could look further into https://github.com/wasp-lang/wasp/pull/1202#discussion_r1204735326 and see what I could find for a workaround.

Reply:
Ha that sucks -> I commented on the issue asking for some ideas on how to go about it, if we could maybe create a PR.

Reply:
Btw there was this comment, referring to (COMMAND | COMMAND) thing that is happening right now: https://github.com/pcapriotti/optparse-applicative/issues/478#issuecomment-1571824106 -> we should also fix that!

Reply:
Idea: what if we put ANSI codes for color, bold and similar, directly into the names of subgroups and commands while providing them to optparse? Ah yes that won't work because it uses those names not only to show them but also to parse them, damn. Ok maybe it would work for subgroups actually hm, but it will certainly not work for commands.